website/www/site/content/en/contribute/_index.md

```diff
@@ -321,69 +367,42 @@ export GOLANG_PROTOBUF_REGISTRATION_CONFLICT=ignore
    all commits in the PR during merge, however if a PR has a mixture of independent changes that should not be squashed, and fixup commits,
    then the PR author should help squashing fixup commits to maintain a clean commit history.
 
-</div>
-
-## When will my change show up in an Apache Beam release?
+#### Apache Beam Releases
 
 Apache Beam makes minor releases every 6 weeks. Apache Beam has a
 [calendar](https://calendar.google.com/calendar/embed?src=0p73sl034k80oob7seouanigd0%40group.calendar.google.com) for
 cutting the next release branch. Your change needs to be checked into master before the release branch is cut
 to make the next release.
 
-## Stale pull requests
+#### Stale Pull Requests
 
 The community will close stale pull requests in order to keep the project
 healthy. A pull request becomes stale after its author fails to respond to
 actionable comments for 60 days.  Author of a closed pull request is welcome to
 reopen the same pull request again in the future. The associated JIRAs will be
 unassigned from the author but will stay open.
 
-## Accounts and Permissions
-
-- [Beam issue tracker (JIRA)](https://issues.apache.org/jira/projects/BEAM/issues):
-  Anyone can access it and browse issues. Anyone can register an account and login
-  to create issues or add comments. Only contributors can be assigned issues. If
-  you want to be assigned issues, a PMC member can add you to the project contributor
-  group.  Email the [dev@ mailing list](/community/contact-us)
-  to ask to be added as a contributor in the Beam issue tracker, and include your ASF Jira username.
-
-- [Beam Wiki Space](https://cwiki.apache.org/confluence/display/BEAM/Apache+Beam):
-  Anyone has read access. If you wish to contribute changes, please create an account and request edit access on the
-  [dev@ mailing list](/community/contact-us) (include your Wiki account user ID).
-
-- Pull requests can only be merged by a
-  [Beam committer](https://home.apache.org/phonebook.html?pmc=beam).
-
-- [Voting on a release](https://www.apache.org/foundation/voting.html): Everyone can vote. Only
-  [Beam PMC](https://home.apache.org/phonebook.html?pmc=beam) members should mark their votes as binding.
+### Troubleshooting
 
-## Communication
+If you run into any issues, check out the [contribution FAQ](https://cwiki.apache.org/confluence/display/BEAM/Contributor+FAQ) or ask on the [dev@ mailing list](/community/contact-us/) or [#beam channel of the ASF slack](/community/contact-us/).
```

Comment:
pls change link to : https://beam.apache.org/community/contact-us/
slack -> Slack

Reply:
done